test/choosetests.jl

```diff
@@ -80,10 +80,52 @@ function choosetests(choices = [])
         prepend!(tests, linalgtests)
     end
 
+    sparsetests = ["sparse/sparse", "sparse/sparsevector"]
+    if Base.USE_GPL_LIBS
+        append!(sparsetests, ["sparse/umfpack", "sparse/cholmod", "sparse/spqr"])
+    end
+    if "sparse" in skip_tests
+        filter!(x -> (x != "sparse" && !(x in sparsetests)), tests)
+    elseif "sparse" in tests
+        # specifically selected case
+        filter!(x -> x != "sparse", tests)
+        prepend!(tests, sparsetests)
```

Comment:
we may want to move those slightly earlier, but probably not before linalg or subarray, this reorders things a lot more than I think we want